src/main/java/io/microconfig/osdf/utils/FileUtils.java

```diff
@@ -72,4 +96,22 @@ public static void createDirectoryIfNotExists(Path dir) {
     public static String hashOfFile(Path path) {
         return md5Hex(readAll(path));
     }
+
+    public static void createFileIfNotExists(Path file) {
+        if (!exists(file)) {
+            try {
+                createFile(file);
+            } catch (IOException e) {
+                throw new RuntimeException("Can't create file " + file);
+            }
+        }
+    }
+
+    public static Stream<Path> getPathsInDir(Path dir) {
```

Comment:
that's an old method

Files.list should be used in try-with-resources block, so if this is really needed modify to return list of paths

Reply:
done